torch/nn/modules/loss.py

```diff
@@ -33,12 +33,13 @@ class L1Loss(_Loss):
     where :math:`N` is the batch size. If reduce is ``True``, then:
 
     .. math::
-        \ell(x, y) = \begin{cases}
-            \operatorname{mean}(L), & \text{if}\; \text{size_average} = \text{True},\\
-            \operatorname{sum}(L),  & \text{if}\; \text{size_average} = \text{False}.
+        \ell(x, y) =
+        \begin{cases}
+            \operatorname{mean}(L), & \text{if size\_average} = \text{True;}\\
+            \operatorname{sum}(L),  & \text{if size\_average} = \text{False.}
         \end{cases}
 
-    `x` and `y` arbitrary shapes with a total of `n` elements each.
+    `x` and `y` represent arbitrary shapes with a total of `n` elements each.
```

Comment:
Thanks for spotting this. I think it should be `` `x` and `y` are tensors of arbitrary ... ``